honeybee_energy/properties/face.py

```diff
@@ -0,0 +1,116 @@
+# coding=utf-8
+"""Face Energy Properties."""
+from ..construction import OpaqueConstruction
+from ..lib.default.room import generic_costruction_set
+
+
+class FaceEnergyProperties(object):
+    """Energy Properties for Honeybee Face.
+
+    Properties:
+        construction
+        materials
+        r_value
+        u_value
+        r_factor
+        u_factor
```

Comment:
should we also expose `thickness` somewhere? I know it is not an energy property per se. 🤔

Reply:
See my comment above.  I think, for the sake of clarity, these should all just be accessed through he construction.

Reply:
sounds good.